### PR TITLE
IG unsent support

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: handlers/facebookapp/facebookapp.go [hunk 1]
```diff
@@ -142,6 +142,7 @@ type moPayload struct {
 				IsEcho      bool   `json:"is_echo"`
 				MID         string `json:"mid"`
 				Text        string `json:"text"`
+				IsDeleted   string `json:"is_deleted"`
```
Comment (rowanseymour) [inline handlers/facebookapp/facebookapp.go:145]: is this definitely correct according to their docs? I notice above that `is_echo` is an actual `bool`? Can go unmarshalling convert `"true"` to `true`?

Comment (norkans7) [reply handlers/facebookapp/facebookapp.go:145]: There they send a string as shown on the webhook events example

https://developers.facebook.com/docs/messenger-platform/instagram/features/webhook#webhook-events

Comment (rowanseymour) [reply handlers/facebookapp/facebookapp.go:145]: Well that's some impressively messy API design on their part

File: backends/rapidpro/backend.go [hunk 2]
```diff
@@ -101,6 +101,28 @@ func (b *backend) RemoveURNfromContact(ctx context.Context, c courier.Channel, c
...
+	msgs_msg
+SET
+	visibility = 'X',
+	text = '',
+	attachments = '{}'
+WHERE
+	msgs_msg.id = (SELECT m."id" FROM "msgs_msg" m INNER JOIN "channels_channel" c ON (m."channel_id" = c."id") WHERE (c."uuid" = $1 AND m."external_id" = $2 AND m."direction" = 'I'))
+RETURNING
+	msgs_msg.id
+`
```
Comment (norkans7) [inline backends/rapidpro/backend.go:115]: Matches the soft deletion by https://github.com/nyaruka/rapidpro/pull/3681

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**